### DISCUSSION TITLE
Focus just one biggest object by one bouding box

### DISCUSSION THREAD

**Discussion by TuyenSD** [ultralytics/yolov5#5915] [Q&A]
Hello Sir, can you help me how to Focus just one biggest object by one bouding box

thanks for your model a lot <3

**Comment by glenn-jocher** [comment]
Sort detections by size and select the largest.

**Reply by TuyenSD** [reply]
can you show code for me, please sir

**Reply by TuyenSD** [reply]
![image](https://user-images.githubusercontent.com/86045192/145206080-7911079e-84d6-4460-a761-9fb187199c88.png)

i was thinking about calculating the maximum area but only came here and no idea anymore

**Reply by TuyenSD** [reply]
![image](https://user-images.githubusercontent.com/86045192/145207214-8b1e1dbb-b7f7-4e1d-b075-a58b60cda0c2.png)
oh i changed max_dect, so what next i need to do to focus largest thing, sir
sorry about this problem

**Reply by TuyenSD** [reply]
Pls help me

**Reply by ys31jp** [reply]
you change the max-det and result will return you one detection. 
if you want area, you can compute from the bounding box of the object.

**Reply by TuyenSD** [reply]
thankk you a lot
